web/src/features/charts/RoundedCard.tsx

```diff
@@ -6,14 +6,14 @@ export const RoundedCard = forwardRef<
   { children: ReactNode; className?: string }
 >(function RoundedCard({ children, className }, reference) {
   return (
-    <div
+    <section
```

Comment:
Would `<article>` be better to use here?

Reply:
Don't think so, in my view article would be used mostly for text content and not visualisations like this that can change (using the time slider).

Reply:
I juat saw this and thought that product card would be similar to this. But it's not a dealbreaker for me so I'll let you decide
<img width="677" alt="image" src="https://github.com/user-attachments/assets/25600e98-4f9d-40e5-aaa6-017e8cbc3737">

Reply:
I think it's safer to keep it as a section for now.